_locales/en/messages.json

```diff
@@ -1742,6 +1742,63 @@
 	"hub_setup_feature_products_button": {
 		"message": "Yes, please!"
 	},
+	"hub_products_title" : {
+		"message": "Ghostery Hub - Try Other Ghostery Products"
+	},
+	"hub_products_header": {
+		"message": "Browse Smarter with Ghostery Products!"
+	},
+	"hub_products_sub_header": {
+		"message": "Stay protected and enhance your online experience with the our full privacy suite."
+	},
+	"hub_products_main_promo_title": {
+		"message": "Mobile Privacy Browser"
+	},
+	"hub_products_main_promo_description": {
+		"message": "Browse ad-and tracker free with our updated mobile browser. Now equipped with a private search engine!"
+	},
+	"hub_products_second_promo_title": {
+		"message": "Ghostery Desktop Browser"
+	},
+	"hub_products_second_promo_description": {
+		"message": "A sleek, fast, and customizable browser equipped with full privacy features and anonymous search engine. Coming soon!"
+	},
+	"hub_products_third_promo_title": {
+		"message": "Ghostery New Tab"
+	},
+	"hub_products_third_promo_description": {
+		"message": "Search privately, view tracker stats, and manage Rewards with our new tab feature. Coming soon!"
+	},
+	"hub_rewards_header_title": {
+		"message": "Try Ghostery Rewards!"
+	},
+	"hub_rewards_header_description": {
+		"message": "Our new Rewards feature offers a powerful, completely private new way to receive real discounts online. Turn it on and check it out!"
+	},
+	"hub_rewards_header_learn_more": {
+		"message": "Learn More"
+	},
+	"hub_rewards_first_icon_title": {
```

Comment:
change `first_icon` and similar to use more specific names: experience, privacy, relevant